lib/grape/dsl/inside_route.rb

```diff
@@ -26,42 +26,56 @@ def self.post_filter_methods(type)
       # Methods which should not be available in filters until the before filter
       # has completed
       module PostBeforeFilter
-        def declared(params, options = {}, declared_params = nil)
+        def declared(passed_params, options = {}, declared_params = nil)
```

Comment:
I actually added this in because I thought it made it a little easier to grok.
When this is called and the declared child params params are passed as an argument, its a bit easier to remember that `passed_params` are always the params passed into the method and `declared_params` are the params that were declared in the `params` block